docs/upgrading/upgrading-your-package.md

```diff
@@ -0,0 +1,542 @@
+# Upgrading your package to 1.0 APIs
+
+Atom is rapidly approaching 1.0. Much of the effort leading up to the 1.0 has been cleaning up APIs in an attempt to future proof, and make a more pleasant experience developing packages.
+
+This document will guide you through the large bits of upgrading your package to work with 1.0 APIs.
+
+## TL;DR
+
+We've set deprecation messages and errors in strategic places to help make sure you don't miss anything. You should be able to get 95% of the way to an updated package just by fixing errors and deprecations. There are a couple of things you need to do to enable all these errors and deprecations.
```

Comment:
> There are a couple of things you need to do to enable all these errors and deprecations

This doesn't make sense to me. Aren't the deprecation warnings enabled by default? It seems like you would want to say: "There are a few basic changes most packages will require to avoid the majority of deprecation warnings." Or something like that.